packages/grafana-ui/src/options/builder/hideSeries.tsx

```diff
@@ -43,14 +43,13 @@ export function addHideFrom(builder: FieldConfigEditorBuilder<HideableFieldConfi
     path: 'hideFrom',
     defaultValue: {
       tooltip: false,
-      graph: false,
+      viz: false,
```

Comment:
There was an issue here @torkelo, otherwise it didn't work for hiding in the vis.

Reply:
yea, I noticed that.

Reply:
Need to dig deeper how to make the default value type safe.